x-pack/test/ingest_manager_api_integration/apis/epm/install_by_upload.ts

```diff
@@ -20,15 +20,40 @@ export default function ({ getService }: FtrProviderContext) {
     path.dirname(__filename),
     '../fixtures/direct_upload_packages/apache_0.1.4.tar.gz'
   );
-  const testPkgKey = 'apache-0.14';
+  const testPkgArchiveZip = path.join(
```

Comment:
Sounds like we might be removing the `tar.gz` functionality, if not should we add some of those packages to test as well?

Reply:
Yes, `tar.gz` will be removed. I added it here at all because the registry code also still supports `tar.gz`, but I think it can be removed in both places soon.